.dockerignore

```
@@ -0,0 +1,3 @@
+build/
```

Some more common build directory names
```suggestion
build/
out/
cmake-build-*/
```